spotguide/dronerepoconfig.go

```diff
@@ -0,0 +1,336 @@
+package spotguide
+
+import (
+	"fmt"
+	"strconv"
+
+	libcompose "github.com/docker/libcompose/yaml"
+	"gopkg.in/yaml.v2"
+)
+
+// nolint
+// droneRepoConfig defines a pipeline configuration.
+type droneRepoConfig struct {
+	cache     libcompose.Stringorslice
+	platform  string
+	branches  droneConstraint
+	workspace droneWorkspace
+	cluster   *droneKubernetesCluster
+	clone     droneContainers
+	pipeline  droneContainers
+	services  droneContainers
+	networks  droneNetworks
+	volumes   droneVolumes
+	labels    libcompose.SliceorMap
+}
+
+// nolint
+// droneKubernetesCluster defines a cluster that has to be created before executing the rest of the Pipeline.
+type droneKubernetesCluster struct {
+	name     string `yaml:"name"`
```

Comment:
I think in order to make a struct serializable using json or yaml, at least the struct field must be exported.

Reply:
Fixed.